src/RA_Interface.h

```diff
@@ -15,22 +15,22 @@ struct ControllerInput
     BOOL m_bDownPressed;
     BOOL m_bLeftPressed;
     BOOL m_bRightPressed;
-    BOOL m_bConfirmPressed;	//	Usually C or A
-    BOOL m_bCancelPressed;	//	Usually B
-    BOOL m_bQuitPressed;	//	Usually Start
+    BOOL m_bConfirmPressed; //	Usually C or A
+    BOOL m_bCancelPressed;  //	Usually B
+    BOOL m_bQuitPressed;    //	Usually Start
 };
 
 enum EmulatorID
 {
-    RA_Gens = 0,
-    RA_Project64 = 1,
-    RA_Snes9x = 2,
+    RA_Gens             = 0,
+    RA_Project64        = 1,
+    RA_Snes9x           = 2,
     RA_VisualboyAdvance = 3,
-    RA_Nester = 4,
-    RA_FCEUX = 5,
-    RA_PCE = 6,
-    RA_Libretro = 7,
-    RA_Meka = 8,
+    RA_Nester           = 4,
+    RA_FCEUX            = 5,
+    RA_PCE              = 6,
+    RA_Libretro         = 7,
+    RA_Meka             = 8,
```

Comment:
This is a good use of aligning equals, but if it means causing the other alignings to happen, I'd prefer to leave it the way it was.